core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -6,9 +6,14 @@
          ng-class="{'fa-ellipsis-h': audioLoadingIndicatorIsShown, 'fa-play': !isAudioPlaying(), 'fa-pause': isAudioPlaying(), 'audio-controls-audio-not-available': !isAudioAvailableInCurrentLanguage() || audioIsLoading}"
          uib-tooltip="<[!isAudioAvailableInCurrentLanguage() ? ('I18N_PLAYER_AUDIO_NOT_AVAILABLE_IN' | translate:{languageDescription:getCurrentAudioLanguageDescription()}) : '']>"
          tooltip-append-to-body="true" tooltip-placement="right"></i>
-      <div class="audio-bar slider-section" ng-if="hasPressedPlayButtonAtLeastOnce">
-        <md-slider ng-model="track.progress" ng-model-options="{ getterSetter: true }" aria-label="audio-slider">
-        </md-slider>
+      <div class="slider-section" ng-if="hasPressedPlayButtonAtLeastOnce">
+        <div ng-if="audioLoadingIndicatorIsShown">
+          <md-progress-linear md-mode="indeterminate"> </md-progress-linear>
```

Comment:
Drop the space between the opening and closing tags.

Reply:
Oh! I'm Sorry for this. My bad!